examples/src/mysql_demo/mysql_storables.rs

```diff
@@ -38,6 +38,7 @@ pub(crate) trait MySqlStorable {
 
     fn set_batch_params(items: &[DbRecord]) -> Result<mysql_async::Params>;
 
+    #[allow(dead_code)]
```

Comment:
Also this one

Reply:
same as above. This is added specifically to address warnings that showed up during CI.